packages/investor-yearn/src/api/api.ts

```diff
@@ -17,6 +17,7 @@ import {
   yv2VaultAbi
 } from '../constants'
 import { bnOrZero, buildTxToSign } from '../utils'
```

Comment:
```suggestion
import { bnOrZero, buildTxToSign, transformVault } from '../utils'
```

Since other utils are exported from `packages/investor-yearn/src/utils/index.ts`, you could also export this one there and use it like so so you don't have to import from `utils` twice.

Reply:
good catch! vscode had done that, fixed now!

Reply:
Sweet! Just need to add it in `packages/investor-yearn/src/utils/index.ts` now:

```shell
[tsserver 2305] [E] Module '"../utils"' has no exported member 'transformVault'.
```

Reply:
oh lol, my bad. no wonder vscode didn't import it from index 🤣